react-validated-input-component/src/validated-input.jsx

```diff
@@ -0,0 +1,91 @@
+import React from 'react';
+
+const specialChars = ['!', '@', '#', '$', '%', '^', '&', '*', '(', ')'];
+
+class ValidInput extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      password: '',
+      valid: false,
+      checkOrX: false,
+      error: ''
```

Comment:
Instead of storing all three of these, try only storing the error message. If it's empty, then the password is valid and you should show the check. otherwise, you should show the ex.